binding.cc

```diff
@@ -9,14 +9,18 @@
 
 #define BUFFER_CAST(type, name, info) \
   type *name = (type *) node::Buffer::Data(info->ToObject());
- 
+
```

Comment:
I can undo this if you like … it’s my editor trying to do its thing. But I would expect a lot of people’s editors to do this :)